conda_forge_tick/update_sources.py

```diff
@@ -590,6 +590,39 @@ def get_url(self, meta_yaml) -> Optional[str]:
         return super().get_url(meta_yaml)
 
 
+class GitTags(AbstractSource):
+    name = "GitTags"
+
+    def get_url(self, meta_yaml):
+        return (
+            (meta_yaml.get("meta_yaml") or meta_yaml.get("recipe_yaml", {}))
```

Comment:
We confusingly always call it "meta_yaml". https://github.com/regro/cf-scripts/blob/main/conda_forge_tick/feedstock_parser.py#L462

Reply:
The test code is setting these keys incorrectly for v1 I think.